src/selection/style.js

```diff
@@ -16,7 +16,10 @@ d3_selectionPrototype.style = function(name, value, priority) {
     }
 
     // For style(string), return the computed style value for the first node.
-    if (n < 2) return d3_window.getComputedStyle(this.node(), null).getPropertyValue(name);
+    if (n < 2) {
+      var node = this.node();
+      return node.ownerDocument.defaultView.getComputedStyle(node, null).getPropertyValue(name);
```

Comment:
I had this exact change in my working tree but I forgot to commit it last night, so thanks!